.github/workflows/check-pep8.yml

```diff
@@ -2,7 +2,8 @@ name: autopep8
 on:
   push:
     branches:
-      - "*"
+      - main
+      - develop
```

Comment:
Is this change actually relevant to this PR? And why not check the formatting for other branches?

Reply:
Sorry, forgot to remove this change. If we run tests on all branches when pushing, the tests are executed twice. One time because we push and the other time because we probably created a pull request.
It is not necessary to make this change, but I think it may makes sense as it reduces computation time. The [main repo](https://github.com/precice/precice/blob/8da078b35efd7fe91838c4996bbb5e7658f3e7bb/.github/workflows/build-and-test.yml#L8-L23) also does it this way.

Reply:
Makes sense, can you please open a separate PR with this change?